internal/telemetry/otelconv/otelconv.go

```diff
@@ -0,0 +1,97 @@
+package otelconv
+
+// This file contains constants for telemetry instrumentation that follow the OpenTelemetry
+// Semantic Conventions for naming. The naming conventions adhere to the guidelines specified at:
+// https://opentelemetry.io/docs/specs/semconv/general/naming/
+//
+// The events and attributes defined here are used throughout the SpiceDB codebase to provide
+// a consistent naming scheme for spans, events, attributes, etc.
+//
+// IMPORTANT: When adding a new _custom_ name, consider the following conventions:
+//  - Start all names with "spicedb" prefix to identify our application domain
```

Comment:
Add a string slice containing all of the event names and all the attributes, and then add a unit test enforcing these rules on all names found